.travis.yml

```diff
@@ -40,6 +40,11 @@ jobs:
         - DESC="compile performance-tests"
         - CMD="mvn install --projects performance-tests --also-make --activate-profiles all -DskipTests=true"
 
+    - jdk: oraclejdk8
+      env:
+        - DESC="compile javadoc"
+        - CMD="mvn install javadoc:jar --projects '!unit-tests,!junit-trait-runner,!unit-tests-java8,!serialization-tests,!scala-unit-tests,!jmh-scala-tests,!jmh-tests,!jcstress-tests,!p2-repository,!test-coverage' --activate-profiles all -DskipTests=true"
```

Comment:
Why activate the all profile, then turn off most projects? I can't keep track of which one is on. Instead of listing negations, why not list the ones you want on?

Reply:
This is the exact build which runs during deploy. I tried doing without negation during deploy but the Jenkins build keeps on failing.